aten/src/ATen/quantized/Quantizer.h

```diff
@@ -59,6 +60,10 @@ struct CAFFE2_API Quantizer : public c10::intrusive_ptr_target {
     return qscheme_;
   }
 
+  virtual ScalarType scalar_type() {
```

Comment:
why is it virtual and not a regular getter?

Reply:
oh, will remove virtual